sitemap.py

```diff
@@ -4,14 +4,15 @@
 import logging
 import re
 import requests
-from datetime import date
+from datetime import *
```

Comment:
I think we should probably not import everything from here?

Reply:
good idea